File: PRAK503-2510817220020-LailyAzizahIslami.c
#include <stdio.h> 
int maksimal(int a, int b){ 
    int maks;
    if (a > b)
    maks = a;

    else 
    maks = b;
    return maks;
} 
int minimal(int a, int b){
    int minim; 
    if (a < b)
    minim = a;

    else 
    minim = b;
    return minim;
}
int main(){ 
    int batas = 0; 
    int maks = -100000; 
    int minim = 100000; 
    int bilangan; 
    scanf("%d", &bilangan); 
    while(batas < bilangan){ 
      int nilai; 
      scanf("%d", &nilai); 
      maks = maksimal(maks, nilai); 
      minim = minimal(minim, nilai); 
      batas++; 
      } 
    printf("%d %d",maks,minim);
}

File: PRAK503-2510817220020-LailyAzizahIslami.py
def maksimal(a, b):
    if a > b:
        return a
    else:
        return b

def minimal(a, b):
    if a < b:
        return a
    else:
        return b

batas = 0
maks = -1000000
minim = 1000000
bilangan = int(input())
nilai = map(int, input().split())

for n in nilai:
    maks = maksimal(maks, n)
    minim = minimal(minim, n)

print(maks, minim)